pandoc/extras/build_image.bat

```diff
@@ -0,0 +1,218 @@
+@rem ## Created by mics (https://github.com/michelou/) on December 2020.
```

Comment:
If image can be built by Linux/MacOS and published, then we don't need the script to build the image on Windows.

Reply:
My wish here is to offer different ways to developers to generate a PDF file from the Markdown files *before* they publish their changes (more details per email).